linux_os/guide/services/ssh/ssh_server/sshd_set_max_sessions/rule.yml

```diff
@@ -0,0 +1,30 @@
+documentation_complete: true
+
+title: 'Set SSH MaxSessions limit'
+
+description: |-
+    The <tt>MaxSessions</tt> parameter specifies the maximum number of open sessions permitted
+    from a given connection.  Setting MaxSessions to 1 will effectively disable
+    session multiplexing, whereas setting it to 0 will prevent all
+    shell, login and subsystem sessions while still permitting forwarding.  The default is 10.
+    Add or correct the following lines to 4 or less. To set MaxSessions edit
+    <tt>/etc/ssh/sshd_config</tt> as follows: <pre>MaxSessions <b>sessions</b></pre>
+
+rationale: |-
+    To protect a system from denial of service due to a large number of concurrent
+    sessions, use the rate limiting function of MaxSessions to protect availability
+    of sshd logins and prevent overwhelming the daemon.
+
+severity: medium
+
+references:
+  cis@rhel8: 5.2.19
+  cis@sle15: 5.2.23
+
+ocil_clause: 'it is commented out or not configured properly'
+
+ocil: |-
+    Run the following command to see what the max sessions number is:
+    <pre>$ sudo grep MaxSessions /etc/ssh/sshd_config</pre>
+    If properly configured, the output should be:
+    <pre>MaxSessions <sub idref="sshd_max_sessions" /></pre>
```

Comment:
Please, make sure all files end with a newline, check the GitHub red marks.
Not an empty line, a newline, :)

Reply:
Resolved in 9800aa6